### ISSUE TITLE
Promote and demote

### ISSUE THREAD

**Issue by danielmatz** [sensorflo/adoc-mode#22]
I'm a little confused by your word choice for `adoc-promote` and `adoc-denote`.

First, the opposite of promote is demote, not denote.  Denote means to indicate.

Second, promote in this sense means to increase in rank.  Shouldn't that take a heading and make it a more important heading?  Right now, it turns a heading into a subheading.

Minor issues, I know, but I thought I'd open an issue anyway.  Thanks for adoc-mode!

**Comment by sensorflo** [comment]
You're right in both points. I corrected the mistake of saying denote of demote and pushed the commit. I am not sure if I now should toggle the meaning of demote and promote, now that the 'wrong' meaning is already out for a few years.

**Comment by nicorikken** [comment]
Had this same remark starting to use `adoc-mode`. The current inverse behavior is just confusing to me.

**Comment by jack99999** [comment]
just found the promote/demote key bindings. definitely backwards. 

incidentally, the wiki home page has two occurrences of 'denote' instead of 'demote'
